src/components/Forms/AccountEditForm/index.tsx

```diff
@@ -234,19 +130,18 @@ export const AccountEditForm = ({
             required
           />
         </StyledInputContainer>
-        {notification && <UserNotification {...notification} />}
-        {children}
+        {errorNotifications &&
+          errorNotifications.map(not => {
```

Comment:
Mehh, this is poor variable naming. Should be `notification` or `errorNotification`